src/components/Person/Person.jsx

```diff
@@ -1 +1,21 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  function isMaried(people) {
```

Comment:
- Move this function outside the component
- Function name should start with a verb. And as it returns a string, it shouldn't be something with `is` as `is` or `has` are used for a boolean variables